package.json

```diff
@@ -46,13 +46,21 @@
     "test-rooms-disabled": "ENABLE_ROOMS=false HIDDEN_ROUTES=/rooms mocha -R dot --recursive tests/roomFlag/roomFlagDisabled",
     "test-db": "npm run checkdb && mocha -R dot --recursive tests/db",
     "test-cors": "CORS_ROUTES=/v1/bots,/v1/botActions,/v1/botData,/v1/events,/v1/rooms mocha -R dot --recursive tests/cors",
-    "test-jobQueue": "mocha --recursive tests/jobQueue",
+    "test-jobQueue": "npm run test-with-coverage -- --recursive tests/jobQueue",
     "test-cache": "npm run test-cache-jobQueue && npm run test-cache-models && npm run test-cache-misc",
-    "test-cache-jobQueue": "mocha --recursive tests/cache/jobQueue",
-    "test-cache-models": "mocha --recursive tests/cache/models",
-    "test-cache-misc": "mocha tests/cache",
+    "test-cache-jobQueue": "npm run test-with-coverage -- --recursive tests/cache/jobQueue",
+    "test-cache-models": "npm run test-with-coverage -- --recursive tests/cache/models",
+    "test-cache-misc": "npm run test-with-coverage -- tests/cache",
     "test-view": "NODE_ENV=build mocha -R dot --recursive --compilers js:babel-core/register --require ./tests/view/setup.js tests/view",
-    "test": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R dot --recursive tests/api tests/clock tests/config tests/db tests/enforceToken tests/realtime tests/enableCache tests/logging && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage && npm run test-cache && npm run test-jobQueue && npm run test-view && npm run test-disablehttp && npm run test-rooms-enabled && npm run test-rooms-disabled && npm run test-cors && npm run test-limiter",
+    "test-main": "npm run test-with-coverage -- --recursive tests/api tests/clock tests/config tests/db tests/enforceToken tests/realtime tests/enableCache tests/logging",
+    "test-extra": "npm run test-cache && npm run test-jobQueue && npm run test-view && npm run test-disablehttp && npm run test-rooms-enabled && npm run test-rooms-disabled && npm run test-cors && npm run test-limiter",
+    "test-all": "npm run test-main && npm run test-extra",
+    "test": "npm run reset-coverage && npm run test-all && npm run report",
+    "test-travis": "npm run test && npm run coveralls",
+    "test-with-coverage": "istanbul cover ./node_modules/mocha/bin/_mocha -report none --print none --include-pid --",
+    "report": "istanbul report text-summary lcovonly",
+    "reset-coverage": "rm -rf ./coverage",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && npm run reset-coverage",
```

Comment:
I made some changes to the test scripts to get the coverage reporting working across multiple mocha runs. Coverage is back up to 88% now.

I could also apply these changes to the test scripts that previously weren't being included in the test coverage. (in test-extra) Any issue with that?

Reply:
No issues from my side.

Reply:
Please do it!!!